src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerView.vb

```diff
@@ -782,12 +782,12 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
                                 '  SpecialFileCustomDocumentMonikerProvider).
                                 .CustomMkDocumentProvider = New SpecialFileCustomDocumentMonikerProvider(Me, __PSFFILEID2.PSFFILEID_AssemblyResource)
                             Else
-                                .CustomViewProvider = New SpecialFileCustomViewProvider(Me, DesignerPanel, __PSFFILEID2.PSFFILEID_AssemblyResource, SR.GetString(SR.APPDES_ClickHereCreateResx))
+                                .CustomViewProvider = New SpecialFileCustomViewProvider(Me, DesignerPanel, __PSFFILEID2.PSFFILEID_AssemblyResource, SR.GetString(My.Resources.Microsoft_VisualStudio_AppDesigner_Designer.APPDES_ClickHereCreateResx))
                             End If
-                        ElseIf System.String.Compare(Microsoft.VisualBasic.Right(FileName, 9), ".settings", StringComparison.OrdinalIgnoreCase) = 0 Then
+                        ElseIf System.String.Compare(VisualBasic.Right(FileName, 9), ".settings", StringComparison.OrdinalIgnoreCase) = 0 Then
                             'Add .settings file with a known editor so user config cannot change
                             .EditorGuid = New Guid(My.Resources.Microsoft_VisualStudio_AppDesigner_Designer.SettingsDesignerEditorFactory_GUID)
-                            .EditorCaption = SR.GetString(SR.APPDES_SettingsTabTitle)
+                            .EditorCaption = SR.GetString(My.Resources.Microsoft_VisualStudio_AppDesigner_Designer.APPDES_SettingsTabTitle)
```

Comment:
There is a comment on 'Microsoft_VisualStudio_AppDesigner_Designer' wanting to avoid the name Microsoft_VisualStudio_AppDesigner_Designer. Check comment on `Partial Friend Class Microsoft_VisualStudio_AppDesigner_Designer`